Cargo.toml

```diff
@@ -35,3 +35,5 @@ regex = "1"
 glob = "0"
 notify = "4"
 shellexpand = "1"
+tokio = { version = "0.2.13", features = [ "rt-core", "rt-threaded", "macros", "blocking" ] }
+futures = { version = "0.3.4", features = [ "thread-pool" ] }
```

Comment:
Why thread-pool feature is needed here?

Reply:
It isn't. I used `FuturesUnordered` before the first changes made to the PR and then it was needed, but it isn't anymore. Removed it!